internal/command/launch/launch.go

```diff
@@ -256,6 +258,16 @@ func run(ctx context.Context) (err error) {
 		appConfig.SetInternalPort(n)
 	}
 
+	env := flag.GetStringSlice(ctx, "env")
+	if len(env) > 0 {
+		vars, err := cmdutil.ParseKVStringsToMap(env)
```

Comment:
failing for a flag parsing error at this point is a bit too late. It has created the app, the database and run scanners; all expensive operations that takes time until reaching this point of deception 😁

Reply:
thanks, moved it earlier